--- refpolicy/config/file_contexts.subs_dist	2018-05-02 09:15:22.936062330 -0400
+++ clip-selinux-policy/config/file_contexts.subs_dist	2017-11-13 09:06:57.006287468 -0500
@@ -15,6 +15,10 @@
 /libx32 /usr/lib
 /sbin /usr/sbin
 /etc/init.d /etc/rc.d/init.d
+/etc/systemd/system /usr/lib/systemd/system
+/run/systemd/system /usr/lib/systemd/system
+/run/systemd/generator /usr/lib/systemd/system
+/run/systemd/generator.late /usr/lib/systemd/system
 /lib/systemd /usr/lib/systemd
 /run/lock /var/lock
 /usr/lib32 /usr/lib
