--- livecd-creator	2011-09-21 15:31:46.000000000 -0400
+++ /usr/bin/livecd-creator	2010-08-15 19:21:15.000000000 -0400
@@ -25,37 +25,6 @@
 
 import imgcreate
 
-class CLIPLiveImageCreator(imgcreate.x86LiveImageCreator):
-
-    def _configure_bootloader(self, isodir):
-        """configure the boot loader"""
-        os.makedirs(isodir + "/isolinux")
-
-        menu = super(CLIPLiveImageCreator,self)._x86LiveImageCreator__find_syslinux_menu()
-
-        super(CLIPLiveImageCreator,self)._x86LiveImageCreator__copy_syslinux_files(isodir, menu,
-                                   super(CLIPLiveImageCreator,self)._x86LiveImageCreator__find_syslinux_mboot())
-
-        isocfg = """default clip
-timeout 0
-prompt 0
-noescape 1
-allowoptions 0
-
-label clip
-  kernel vmlinuz0
-  append initrd=initrd0.img root=CDLABEL=clip rootfstype=iso9660 ro quiet liveimg live_ram vga=0x317 mem=8704M 
-"""
-
-	# these functions have side effects we need, such as copying the kernel image :-/
-        super(CLIPLiveImageCreator,self)._x86LiveImageCreator__get_image_stanzas(isodir)
-        super(CLIPLiveImageCreator,self)._get_isolinux_stanzas(isodir)
-
-        cfgf = open(isodir + "/isolinux/isolinux.cfg", "w")
-        cfgf.write(isocfg)
-        cfgf.close()
-
-
 class Usage(Exception):
     def __init__(self, msg = None, no_error = False):
         Exception.__init__(self, msg, no_error)
@@ -136,7 +105,7 @@
 
     ks = imgcreate.read_kickstart(options.kscfg)
 
-    creator = CLIPLiveImageCreator(ks, name, fs_label)
+    creator = imgcreate.LiveImageCreator(ks, name, fs_label)
     creator.tmpdir = options.tmpdir
     creator.skip_compression = options.skip_compression
     creator.skip_minimize = options.skip_minimize
