diff --git a/usr/lib/python2.7/site-packages/pylorax/treebuilder.py b/support/treebuilder.py
index 0ae1bc3..c2b6ef5 100644
--- a/usr/lib/python2.7/site-packages/pylorax/treebuilder.py
+++ b/support/treebuilder.py
@@ -139,11 +139,17 @@ class RuntimeBuilder(object):
     def generate_module_data(self):
         root = self.vars.root
         moddir = joinpaths(root, "lib/modules/")
+
+        # Generate_module_data creates a file called "module-info" in this
+        # directory. If we don't do something to exclude this file, depmod will fail
+        # on the second path of this loop. Let's check to see if kver is a directory 
+        # before generating module info from it.
         for kver in os.listdir(moddir):
-            ksyms = joinpaths(root, "boot/System.map-%s" % kver)
-            logger.info("doing depmod and module-info for %s", kver)
-            runcmd(["depmod", "-a", "-F", ksyms, "-b", root, kver])
-            generate_module_info(moddir+kver, outfile=moddir+"module-info")
+            if os.path.isdir(kver):
+                ksyms = joinpaths(root, "boot/System.map-%s" % kver)
+                logger.info("doing depmod and module-info for %s", kver)
+                runcmd(["depmod", "-a", "-F", ksyms, "-b", root, kver])
+                generate_module_info(moddir+kver, outfile=moddir+"module-info")
 
     def create_runtime(self, outfile="/var/tmp/squashfs.img", compression="xz", compressargs=[], size=2):
         # make live rootfs image - must be named "LiveOS/rootfs.img" for dracut
