diff -urN a/imgcreate/live.py b/imgcreate/live.py
--- a/imgcreate/live.py	2013-07-09 17:00:56.052825287 -0400
+++ b/imgcreate/live.py	2013-07-09 17:04:24.687772005 -0400
@@ -187,7 +187,8 @@
     def _mount_instroot(self, base_on = None):
         LoopImageCreator._mount_instroot(self, base_on)
         self.__write_initrd_conf(self._instroot + "/etc/sysconfig/mkinitrd")
-        self.__write_dracut_conf(self._instroot + "/etc/dracut.conf")
+        # SRS: fix for https://bugzilla.redhat.com/show_bug.cgi?id=766955
+        self.__write_dracut_conf(self._instroot + "/etc/dracut.conf.d/02livecd.conf")
 
     def _unmount_instroot(self):
         self.__restore_file(self._instroot + "/etc/sysconfig/mkinitrd")
@@ -267,6 +268,8 @@
         f = open(path, "a")
         f.write('filesystems+="' + self.__extra_filesystems() + ' "\n')
         f.write('drivers+="' + self.__extra_drivers() + ' "\n')
+        # SRS: fix for https://bugzilla.redhat.com/show_bug.cgi?id=766955
+        f.write('add_dracutmodules+=" dmsquash-live clip "')
         f.close()
 
     def __create_iso(self, isodir):
