diff -urN a/imgcreate/kickstart.py b/imgcreate/kickstart.py
--- a/imgcreate/kickstart.py	2013-07-09 22:46:05.014957922 -0400
+++ b/imgcreate/kickstart.py	2013-07-09 23:02:22.770584144 -0400
@@ -477,7 +477,9 @@
         return default
     return int(ks.handler.bootloader.timeout)
 
-def get_kernel_args(ks, default = "ro liveimg quiet"):
+# SRS: drop quiet since we really want noisy boot info
+#def get_kernel_args(ks, default = "ro liveimg quiet"):
+def get_kernel_args(ks, default = "ro liveimg live_ram"):
     if not hasattr(ks.handler.bootloader, "appendLine"):
         return default
     if ks.handler.bootloader.appendLine is None:
diff -urN a/imgcreate/live.py b/imgcreate/live.py
--- a/imgcreate/live.py	2013-07-09 22:46:05.015957737 -0400
+++ b/imgcreate/live.py	2013-07-09 22:59:27.726116135 -0400
@@ -103,9 +103,13 @@
         """
         r = kickstart.get_kernel_args(self.ks)
         if os.path.exists(self._instroot + "/usr/bin/rhgb"):
-            r += " rhgb"
+            # SRS: disable RHGB because it is unreliable across a variety of vid cards
+            #r += " rhgb"
+            r += ""
         if os.path.exists(self._instroot + "/usr/bin/plymouth"):
-            r += " rhgb"
+            # SRS: disable RHGB because it is unreliable across a variety of vid cards
+            #r += " rhgb"
+            r += ""
         return r
         
     def _get_mkisofs_options(self, isodir):
@@ -612,9 +616,10 @@
                                                timeout = self._timeout * 10)
 
         cfg += self.__get_image_stanzas(isodir)
-        cfg += self.__get_memtest_stanza(isodir)
-        cfg += self.__get_local_stanza(isodir)
-        cfg += self._get_isolinux_stanzas(isodir)
+        # SRS: remove unused menu opts
+        #cfg += self.__get_memtest_stanza(isodir)
+        #cfg += self.__get_local_stanza(isodir)
+        #cfg += self._get_isolinux_stanzas(isodir)
 
         cfgf = open(isodir + "/isolinux/isolinux.cfg", "w")
         cfgf.write(cfg)
