diff --git a/RHEL6/transforms/xccdf-addfixes.xslt b/RHEL6/transforms/xccdf-addfixes.xslt
index 26a14bd..810477e 100644
--- a/RHEL6/transforms/xccdf-addfixes.xslt
+++ b/RHEL6/transforms/xccdf-addfixes.xslt
@@ -18,7 +18,7 @@
         <xsl:if test="@rule=$rule_id">
           <xsl:element name="fix" namespace="http://checklists.nist.gov/xccdf/1.1">
           <xsl:attribute name="system"><xsl:value-of select="$fixsystem"/></xsl:attribute>
-          <xsl:value-of select="text()"/>
+          <xsl:copy-of select="node()"/>
           </xsl:element>
         </xsl:if>
       </xsl:for-each> 
